packages/Takeover/stories/Takeover.stories.js

```diff
@@ -80,21 +82,22 @@ storiesOf("Takeover", module)
       );
     })
   )
+
   .add("with nested Popover", () => (
     <TakeoverStory>
       <Takeover.Content>
         <p>
           This example demonstrates how the Popover handles focus when created inside of a Takeover. Click the icons
           below to see how the popovers behave.
         </p>
-        <Popover>
+        <Popover zIndex={11}>
```

Comment:
Technically `6` would work here.  Again, I think we need a deliberate set of guidelines for what goes on what level... but because the Takeover is already at a certain level, a Popover "inside" a Takeover may need to be on a higher... uh... plane?